### DISCUSSION TITLE
Host groups?

### DISCUSSION THREAD

**Discussion by rkelleyrtp** [lxdware/lxd-dashboard#31] [Ideas]
After working with LXDWare for a while, I discovered the container listing is either for a specific, standalone host or a cluster of hosts.  In our deployment, we have a number of standalone hosts that manage a group of containers. In our case, we have three groups of LXD servers:  Admin Servers, QA Servers, and Prod Servers.  We would like to get a list of containers per group.

That said, is it possible to create a new concept named Host Groups that allow you to include individual hosts as part of a group?  

Note: we did not deploy LXD clustering due to the large number of bugs introduced with the new feature.  We tried using the cluster option and many of our LXD Servers had issues (LXD failed to start due to cluster DB problems, etc).

**Comment by matthewalanpenning** [comment]
This is an interesting idea. It is not an LXD specific feature, but may be something I could include on the hosts page. My original design thoughts would be to have three separate LXD Dashboards running one for each environment: Admin, QA, and Prod. And then each dashboard would connect to all the LXD hosts within that environment.

In the meantime, if you wanted to just use a single LXD Dashboard, an idea may be to use a prefix in the alias names for each of your environments such as qa-server1.

**Reply by rkelleyrtp** [reply]
Thanks Matt.  Using prefix names is exactly what we do now (eg: Admin-Server-01,Admin-Server-02, Admin-Server-03, QA-Server-01, QA-Server-02, QA-Server-03,).  I have all the hosts loaded into the dashboard but the containers are only visible from their respective hosts.  The issue is we can only see containers that belong to specific host.  We would like to be able to see all containers from hosts in their group.

My suggestion:  In the Remote Hosts "add" screen, provide the capability to create a group object, and then use tags to connect a host to a group.  Next, put the group name in the same level as the Host name (left pane) with all the same functions (Containers, Virtual Machines, Images, Profiles, etc).
